server/src/domain/person/person.service.ts

```diff
@@ -80,6 +83,104 @@ export class PersonService {
     };
   }
 
+  async createPerson(authUser: AuthUserDto, dto: AssetFaceUpdateDto): Promise<PersonResponseDto> {
+    const changeFeaturePhoto: string[] = [];
+
+    const newPerson = await this.repository.create({ ownerId: authUser.id });
```

Comment:
It is possible to create a new person without the "person_create" permission. It looks like it is possible to create a person without actually linking it to anything.

Should we do the linking here in the server? Would it be better for the client to call a separate endpoint to re-assign a face to the newly created person? Sometimes when the server touches lots of resources like this it becomes not very restful and makes it a bit overly complicated IMO.